packages/cockpit/src/transactions/details/result.js

```diff
@@ -344,8 +345,8 @@ const buildRecipients = applySpec({
 const hasSplitRules = propSatisfies(complement(isEmpty), 'split_rules')
 
 /* eslint-disable-next-line camelcase */
-const groupInstallments = ({ payables, split_rules }) =>
-  split_rules.map(rule => ({
+const groupInstallments = ({ payables, split_rules }) => split_rules
```

Comment:
que tal aqui usar um rename no destructuring pra evitar o `* eslint-disable-next-line camelcase`?
```suggestion
const groupInstallments = ({ payables, split_rules: splitRules }) => splitRules
```